content/coop.html

```diff
@@ -128,23 +128,23 @@ <h3> Salary </h3>
 
 <div class="vbx2"></div>
 
-<h3> Coop Applications Breakdown</h3>
+<h3> Co-op Applications Breakdown</h3>
 <p class="full">
   <span class="big">
-  As respondents gained coop experience, they applied to less jobs over time
+  As respondents gained work experience, they applied to less jobs over time
   and found more jobs outside of Waterlooworks.
   </span>
-  Respondents applied to 3 times less jobs by their last coops. The number of interviews
-  respondents had every term varied between 3.62 and 8.20 and the number of offers received
+  Respondents applied to 3 times less jobs by their last co-ops. The number of interviews
+  respondents had every term varied between 3.62 and 8.20, and the number of offers received
   every term varied between 1.6 to 2.84.
   <br /><br />
   Around 30% of respondents found their job externally. In particular, 60% of respondents found their
-  job externally on their last coop term. There could be several reasons for this increase, the first one
+  job externally on their last co-op term. There could be several reasons for this increase, the first one
```

Comment:
```suggestion
  job externally on their last co-op term. There could be several reasons for this increase: the first one
```

(again, since i saw it here, adding the suggestion commit)